command/server/seal/server_seal.go

```diff
@@ -11,36 +10,30 @@ import (
 )
 
 var (
-	ConfigureSeal func(*server.Config, *[]string, *map[string]string, log.Logger, vault.Seal) (vault.Seal, error) = configureSeal
+	ConfigureSeal func(*server.Seal, *[]string, *map[string]string, log.Logger, vault.Seal) (vault.Seal, error) = configureSeal
```

Comment:
My code editor complains that the func can be fully omitted here; it prefers instead just `ConfigureSeal = configureSeal`.

Reply:
Done